application/app/Models/Product.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Product extends Model
+{
+    //
+    protected $table = 'products';
+
+    protected $fillable = ['image_path', 'name', 'price', 'description', 'product_category_id'];
```

Comment:
```suggestion
    protected $fillable = [
        'image_path',
        'name',
        'price',
        'description',
        'product_category_id',
    ];
```

みやすさ意識で改行してください。
末尾の要素にも`,` 入れておくと追加のときに手間がかかりません。

Reply:
承知しました。